app/v2/models/request/MessageType.scala

```diff
@@ -95,12 +95,11 @@ object MessageType {
   )
 
   val messageTypesSentByArrivalTrader: Seq[MessageType] = Seq(
-    ArrivalNotification,
-    UnloadingRemarks,
-    InformationAboutNonArrivedMovement
+    UnloadingRemarks
```

Comment:
An ArrivalNotification is sent by the trader. It's the first message they send to start an arrival movement

Reply:
I have moved this to messageTypesSentToArrivalTrader Seq see below. Same is done for departure DeclarationData. As with this endpoint we need to first validate messageType using XmlParsrs messageTypeExtractor and ArriavlNotification is not correct messageType for this endpoint so I have moved out to messageTypesSentToArrivalTrader Seq

Reply:
Which is wrong -- because `ArrivalNotifiaction` is _not_ sent to the trader. It was in the right place.

I see `DeclarationData` is in `messageTypesSentToDepartureTrader`... that's wrong too. I don't know _why_ it's there, but that needs to be fixed. I get why you want to do this, but we need to do this right (for both arrivals and departures)

Reply:
done the changes for both arrival and departure